services/cert-manager/1.9.1/root-ca/root-ca.yaml

```diff
@@ -1,6 +1,5 @@
 # a quick solution to fix kommander-traefik and kube-oidc-proxy certificate in attached clusters (with and without cert-manager pre-installed)
-# https://jira.d2iq.com/browse/D2IQ-84510
----
+# https://jira.d2iq.com/browse/D2IQ-84510---
```

Comment:
@bcmendoza who/what is optimizing these? can we make our automation little bit less smart :see_no_evil: 
nbd and prolly this happens only once but just curious

Reply:
I think we still need to manually fix this for now